### DISCUSSION TITLE
Extensions used for theme file

### DISCUSSION THREAD

**Discussion by qwert-f** [fcitx5-android/fcitx5-android#182] [General | 讨论]
Is there a guide to customize the theme

**Comment by rocka** [comment]
The "theme" only supports changing color scheme and background image for now, no customizing keyboard layout or behavior.

You can use this online theme designer https://fcitx5-android.github.io/theme-designer/ (repo: https://github.com/fcitx5-android/theme-designer) to edit and preview theme colors, and export to .zip file. But as of now, it does not support background image.
